google/cloud/appengine/CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ include(GoogleCloudCppLibrary)
 
 set(GOOGLE_CLOUD_CPP_SERVICE_DIRS "" "v1/")
 
-google_cloud_cpp_add_ga_grpc_library(appengine "App Engine Admin API")
+google_cloud_cpp_add_ga_grpc_library(appengine "App Engine Admin API"
+                                     SHARED_PROTO_DEPS "logging_type")
```

Comment:
I see, `logging_type` is generated here.

Reply:
yeah, it will be generated by whatever comes first in the alphabet, pretty much.

If you `-DGOOGLE_CLOUD_CPP_ENABLE=logging`, then the `google/cloud/logging` subdirectory will be responsible for generating `logging_type`.